config/spotbugs-exclude.xml

```diff
@@ -45,6 +45,54 @@
       <Method name="main" />
       <Bug pattern="DM_EXIT" />
     </Match>
+    <Match>
+        <!-- this is a JDK design weakness that we can do nothing about -->
```

Comment:
https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/Main.java#L423-L425

Why it is JDK weakness ?
it is problem of spotbugs tool, it should not report issues on this such code.
We need to report issue on spotbugs and suppress until issue on their side is resolved.

comment should be `till LINK_TO_ISSUE`

Reply:
we already experienced this problem, here is our team member post - https://github.com/spotbugs/spotbugs/issues/259#issuecomment-445460892

Reply:
comment should be `till https://github.com/spotbugs/spotbugs/issues/259`

Reply:
done.